tests/models/edx/test_base.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for the base event model"""
+
+import pytest
+from pydantic.error_wrappers import ValidationError
+
+from tests.fixtures.edx.server import BaseEventFactory
+from tests.fixtures.logs import event_factory
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {"ip": "51.255.9.27"},
+        {"ip": "8.8.8.8"},
+        {"ip": ""},
+        {"event_source": "server"},
+        {"agent": ""},
+        {"host": ""},
+        {"referer": ""},
+        {"referer": "https://www.fun-mooc.fr/"},
+        {"referer": "https://www.fun-mooc.fr/event?event_type=page_close"},
+        {"context_": {"user_id": ""}},
+        {"context_": {"user_id": None}},
+        {"context_": {"user_id": 213456}},
+        {"context_": {"org_id": "", "course_id": ""}},
+        {"context_": {"org_id": "org", "course_id": "course-v1:org+any+any"}},
+    ],
+)
+def test_valid_base_event(kwargs):
```

Comment:
Think of test naming pattern as a namespace:

```suggestion
def test_models_edx_base_event_with_valid_content(kwargs):
```

Reply:
Thanks) good point) updated